docs/static_landing_page/assets/img/socials/icon_twitter.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="28" height="23" viewBox="0 0 28 23" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Do we need to have all of these in the repo? In the sphinx theme we are pulling them from somewhere, I think fontawesome?

Reply:
Will modify all Icons after confirming to all Iconfont.